lib/ast/processor.rb

```diff
@@ -1,266 +1,12 @@
 module AST
-  # Processor is a class which helps transforming one AST into another.
-  # In a nutshell, the {#process} method accepts a {Node} and dispatches
-  # it to a handler corresponding to its type, and returns a (possibly)
-  # updated variant of the node.
+  # This class includes {AST::Processor::Process}; however, it is no
+  # longer in use, since the module defines all of the behaviors that
```

Comment:
It is in use, of course, by all the current API consumers. It is deprecated however.